### PR TITLE
PLANNER-2029: Use OptaPlanner logo for dark backgrounds

### PR DESCRIPTION
**Pull request by yurloc**
The color is #D2D2D2 from
https://www.patternfly.org/v4/design-guidelines/styles/colors#color-palette
and matches the color of navigation bar links.

## Current state (alpha)
Selection_006
Basic OptaPlanner logo. Not ideal for dark backgrounds.

## This PR proposal (gamma)
Selection_008
Modification of the alternative OptaPlanner logo for dark backgrounds. This variant uses a shade of grey that matches navigation links' color.

## Alternative solution (beta)
Selection_007
Official alternative OptaPlanner logo for dark backgrounds without modifications. Uses white color.

### CODE REVIEW

**Comment by yurloc** [comment]
@ge0ffrey I've updated PR description and included screenshots of all proposals that we have.

Let's discuss the file name later and now focus on the color. Proposals:
- Alpha) Status quo (original logo).
- Beta) Official logo for dark backgrounds.
- Gamma) Modified logo using a shade of grey from PF palette.

My comments:
- -1 for Alpha (status quo). Looks bad. We have an alternative logo for such situations.
- +1 for Beta proposal. Looks nice. A bit too flashy in my opinion, attracting too much attention. I admit this is a matter of taste.
- +1.5 for Gamma proposal. I find it a bit nicer than Beta.

@ge0ffrey Your thoughts on Beta vs. Gamma? I can go with either of them.

**Comment by ge0ffrey** [comment]
Agreed that Beta is better than alpha. Let's do that discussion on Gamma with Jason and James. Sending mail now :)

**Comment by yurloc** [comment]
## Current state (alpha)
Selection_006

Give :+1: here to vote for this option.

**Comment by yurloc** [comment]
## This PR proposal (gamma)
Selection_008
Modification of the alternative OptaPlanner logo for dark backgrounds. This variant uses a shade of grey that matches navigation links' color.

Give :+1: here to vote for this option.

**Comment by yurloc** [comment]
## Alternative solution (beta)
Selection_007

Give :+1: here to **vote for this option**.

**Comment by yurloc** [comment]
## This PR proposal (gamma)
Selection_008

Modification of the alternative OptaPlanner logo for dark backgrounds. This variant uses a shade of grey from **PatternFly** color palette that matches navigation links' color.

Give :+1: here to **vote for this option**.

**Comment by rsynek** [comment]
@yurloc if you decide to go with "Beta" option, is it possible to change the color of PF navigation links to white, so that the color of the logo and the links matches?

**Comment by yurloc** [comment]
@rsynek No. I'd rather keep the defaults. I think it's fine the way it is and that PatternFly have decided not to use white for a reason.

**Comment by yurloc** [comment]
Proposal beta won. Going with the original, unmodified OptaPlanner logo variant for dark background from https://design.jboss.org/blip/images/optaplanner/optaplanner_logo_dk_200px.png (white color).

@ge0ffrey please review the file name. I used the file from design.jboss.org without modifying it and I'm OK with keeping it like that.

Advantages:
- No modification => it's clear that also the file content doesn't differ from the original.
- Underscores => it's easier to read.

Disadvantages:
- `dk` => not super clear that it means "for dark backgrounds".

**Comment by yurloc** [comment]
Jenkins failure:

Trying to pull repository docker.io/cypress/included ... 

/usr/bin/docker-current: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit.

**Review by ge0ffrey — APPROVED** [review]

File: optaweb-vehicle-routing-frontend/src/ui/header/Header.tsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ import NavigationWithRouter from './Navigation';
 
 const Header: React.FC = () => (
   <PageHeader
-    logo={<Brand src="./assets/images/optaPlannerLogo200px.png" alt="OptaPlanner Logo" />}
+    logo={<Brand src="./assets/images/optaplanner_logo_dk_200px.png" alt="OptaPlanner Logo" />}
```
Comment (ge0ffrey) [inline optaweb-vehicle-routing-frontend/src/ui/header/Header.tsx:23]: Nitpick: given the logo names here https://github.com/kiegroup/optaplanner/tree/master/optaplanner-docs/src/main/asciidoc/shared
and presuming that optaplanner_logo_dk_200px.png is the same as optaPlannerLogoDarkBackground at a different resolution, I would maybe expect optaPlannerLogoDarkBackground200px.png

Comment (yurloc) [reply optaweb-vehicle-routing-frontend/src/ui/header/Header.tsx:23]: OK, aligning with optaplanner-docs.

**Pull request merged**